e2e/settings.spec.ts

```diff
@@ -326,9 +327,9 @@ test.describe('Settings', () => {
 
     await expect(saveButton).not.toBeDisabled();
```

Comment:
Looking at the linked run, I'm now a bit worried that about following situation:
- Test fails after saving the initially changed values.
- All retries will fail, because entering the same _initial_ values will not enable save button.

What do you think about dropping this line? Would it be very weird, to just click the save button at this point, without caring if it's enabled?

Reply:
The only approach I know of that avoids this problem is to use `beforeEach`  to setup the app state for the test. In this case, that means seeding the db.

It slows each test down a bit, but that's fine if it doesn't happen before too many tests.

I'll reorganise the tests a little and do that.